usr/share/lib/img_proof/tests/conftest.py

```diff
@@ -146,31 +147,47 @@ def f():
 
 
 @pytest.fixture()
-def is_suma_server(host, get_baseproduct):
+def is_suma_server(host, get_baseproduct, get_suma_version):
     def f():
         suma_server_product = '/etc/products.d/SUSE-Manager-Server.prod'
         suma_server = host.file(suma_server_product)
         base_product = get_baseproduct()
-
+        suma_version = get_suma_version(suma_server_product)
+
+        if suma_version and suma_version.startswith('4'):
+            # For suma 4.3 baseproduct HAS to be SUMA server
+            return all([
+                suma_server.exists,
+                suma_server.is_file,
+                base_product == suma_server_product
+            ])
+        # For suma >=5 baseproduct is Micro and suma is an additional product
         return all([
-            suma_server.exists,
-            suma_server.is_file,
-            base_product == suma_server_product
+                suma_server.exists,
+                suma_server.is_file,
```

Comment:
In this case we want to check that the baseproduct link points to Micro to make sure we didn't accidentally build an image that has SLES or some other distribution that might get developed at some point in the future.

Reply:
Fixed! I included the check for the baseproduct to be micro for SUMA >= 5.